infrastructure/main.tf

```diff
@@ -53,12 +53,14 @@ locals {
 }
 
 module "db" {
-  source              = "git@github.com:hmcts/moj-module-postgres.git?ref=master"
+  source              = "git@github.com:hmcts/moj-module-postgres?ref=cnp-449-tactical"
```

Comment:
.git is missing here `git@github.com:hmcts/moj-module-postgres.git?ref=cnp-449-tactical`?

Reply:
I also noticed it in the guideline, but decided to copy-paste it and see if the pipeline would break in preview. As the preview pipelines are all breaking now, I can't see that, yet.